### ISSUE TITLE
zig cc -g -gcodeview does not prodcue a PDB file

### ISSUE THREAD

**Issue by niklas-ourmachinery** [ziglang/zig#8407]
I would expect that running `zig cc` with the flags `-g` and `-gcodeview` would produce a PDB file, but the only output I get is the exe file `a.out`.

~~~
C:\work\zig-plugin\test>type test.c
int main() {}

C:\work\zig-plugin\test>zig cc -g -gcodeview test.c

C:\work\zig-plugin\test>dir
 Volume in drive C is Local Disk
 Volume Serial Number is 9CA7-0D6F

 Directory of C:\work\zig-plugin\test

04/01/2021  01:41 PM    <DIR>          .
04/01/2021  01:41 PM    <DIR>          ..
04/01/2021  01:41 PM           816,640 a.out
04/01/2021  01:40 PM                13 test.c
               2 File(s)        816,653 bytes
               2 Dir(s)  38,481,195,008 bytes free
~~~

**Comment by andrewrk** [comment]
Thanks for the report. Fixed in 09000c3f77a85b1b1bf0e19b09aae7deaf9f1faa. Note that on Windows, Zig makes `-gcodeview` is the default when you supply `-g`. I'm a little confused why an `a.out` file is being produced, since we have this logic:

```zig
    const a_out_basename = switch (object_format) {
        .pe, .coff => "a.exe",
        else => "a.out",
    };
```

Looks like that was introduced in 6dfe9cc83ea75a663b4f6dfa488b95950207a1cf, so I'm guessing you're on 0.7.1. Anyway if you want the fix for the pdb files, the CI should have a new zig tarball available for you on the download page within a couple hours.

**Comment by niklas-ourmachinery** [comment]
Great, thanks so much for the quick response!
